.github/workflows/auto_label.yml

```
@@ -0,0 +1,44 @@
+name: Label PRs & Issues
+
+on:
+  issues:
+    types: [opened, edited]
+  pull_request_target:
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  auto-label-rocm:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Retrieve information
+      id: vars
+      run: |
+        set -eux
+        IS_PR=${{ github.event.pull_request }}
+        if [[ -n "${IS_PR}" ]]; then
+          TITLE="${{ github.event.pull_request.title }}"
+          ISSUE_NUMBER=${{ github.event.pull_request.number }}
+        else
+          TITLE="${{ github.event.issue.title }}"
+          ISSUE_NUMBER=${{ github.event.issue.number }}
+        fi
+        echo ::set-output name=TITLE::${TITLE}
+        echo ::set-output name=ISSUE_NUMBER::${ISSUE_NUMBER}
```

These should have quotes I think:
```suggestion
        echo ::set-output name=TITLE::"${TITLE}"
        echo ::set-output name=ISSUE_NUMBER::"${ISSUE_NUMBER}"
```